src/shell/autoexec.cpp

```diff
@@ -636,7 +636,7 @@ std::vector<std_fs::path> AutoExecModule::GetMountableImagesPaths(const std_fs::
 		if (entry.exists() && entry.is_regular_file() ) {
 			auto const& path = entry.path();
 			std::string ext = path.extension();
-			std::ranges::transform(ext, ext.begin(), ::tolower);
+			lowcase(ext);
```

Comment:
Right, so you've discovered it... I'm commenting commit by commit, so ignore my earlier comment please 😄